public/js/reducers/purchase.js

```diff
@@ -1,9 +1,9 @@
 import * as actionTypes from 'constants/action-types';
 
 
-export const initialState = {
+export const initialPurchaseState = {
   completed: false,
-  payment_methods: [],
+  payMethods: null,
```

Comment:
Because there's a difference between not having any payMethods e.g. `[]` and not having run the code to get them `null`. I'm not using that distinction yet but I was thinking it might be necessary if getPayMethods was needed in the future in addition to the way we get them at sign-in currently.